### DISCUSSION TITLE
Paralel plots in windows command line

### DISCUSSION THREAD

**Discussion by tanbosin** [Chia-Network/chia-blockchain#6023] [Support]
How do I create plots in parallel using command line with windows?
The creation of a plot I do, but several in parallel I could not find an example in the documentation.
Thanks

**Comment by tiny0little** [comment]
you need this command ```chia plots create -h```

for example
```chia plots create -n 11 -k 32 -a <fingerprint of your key> -t /media/SSD-2TB-012/CHIA.tmp/ -d /media/HDD-18TB-005/CHIA/```

run few of them in parallel

**Reply by tanbosin** [reply]
and the time interval between the plots?

**Reply by tiny0little** [reply]
it is up to your hardware, you can run them at the same time or with a gap

**Reply by tanbosin** [reply]
no. If I create a new plot with one in construction the first stop.

**Comment by BrandtH22** [comment]
@tanbosin , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
